cmd/runtimecfg/node-ip.go

```diff
@@ -206,10 +208,15 @@ func checkAddressUsable(chosen []net.IP) (err error) {
 }
 
 func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType string) (chosen []net.IP, matchesVips bool, err error) {
+	// timerLoop will hold a time in Seconds to be used with time.Sleep() before going
+	// for the next loop interation.
+	timerLoop := 0
```

Comment:
We may not want to start this at 0. Even on the first retry we want there to be _some_ delay.